frontend/src/components/ThemedButtons/DefenceConfigurationRadioButton.css

```diff
@@ -0,0 +1,30 @@
+.defence-radio-button {
+	position: relative;
+	cursor: pointer;
+	display: inline-block;
+}
+
+.defence-radio-button input {
+	opacity: 0;
```

Comment:
This is unfortunately also hiding the focus indicator when the user tabs through the buttons.